.github/workflows/ci.yml

```
@@ -71,11 +71,15 @@ jobs:
             matrix:
                 os: [ubuntu-latest]
                 node: [23.x, 22.x, 21.x, 20.x, 18.x, "18.18.0"]
+                NODE_OPTIONS: [""]
                 include:
                     - os: windows-latest
                       node: "lts/*"
                     - os: macOS-latest
                       node: "lts/*"
+                    - os: ubuntu-latest
+                      node: 22.x
+                      NODE_OPTIONS: "--experimental-transform-types"
```

Is this correct? Or should it be:

```suggestion
                      NODE_OPTIONS: "--experimental-strip-types"
```

---

I think this is correct. `--experimental-strip-types` is enabled implicitly on Node.js 23.x. An additional environment is only required to test `--experimental-transform-types`, because that flag isn't enabled by default on any version of Node.js at the moment.

---

Gotcha. Can we add a comment to that effect?

---

Done!